src/gauss_pdf_and_cdf.py

```diff
@@ -0,0 +1,99 @@
+import numpy as np
+import plotly.graph_objects as go
+from sympy import Symbol, pi, E, sqrt, integrate, oo
+from abc import ABC, abstractmethod
+from scipy import stats
+import math
+
+
+class ProbabilisticFunction(ABC):
+    @abstractmethod
+    def __init__(self):
+        pass
+
+
+class Visualizer(ProbabilisticFunction):
+    @staticmethod
+    def visualize(x, y, title):
+        fig = go.Figure()
+        fig.add_trace(go.Scatter(x=x, y=y, name=title))
+        fig.update_xaxes(title="x")
+        fig.update_yaxes(title="y")
+        fig.update_layout(title=title)
+        fig.show()
+
+
+class GaussianProbabilisticSetting(ProbabilisticFunction):
+    def __init__(self):
```

Comment:
insert a line between class and its method (in this case, __init__) like:

```py
class GaussianProbabilisticSetting

    def __init__():
```

This is the same case with all other classes